contrib/tablefunc/sql/tablefunc.sql

```diff
@@ -204,3 +204,95 @@ INSERT INTO connectby_int VALUES(111,11);
 INSERT INTO connectby_int VALUES(1,111);
 -- this should not fail due to recursion detection
 SELECT * FROM connectby('connectby_int', 'keyid', 'parent_keyid', '11', 0, '-') AS t(keyid int, parent_keyid int, level int, branch text);
+
+
+-- test for CTAS and DML operations
```

Comment:
Looks like test covers only 3 functions `crosstab(text,text)`, `crosstab(text)`, `connectby(text,text,text,text,integer,text)`.
Is it ok that other functions are not checked?
e.g.
```sql
create table ct2 as SELECT * FROM crosstab2('SELECT rowid, attribute, val FROM ct where rowclass = ''group1'' and (attribute = ''att2'' or attribute = ''att3'') ORDER BY 1,2;');
# ERROR:  function cannot execute on a QE slice because it accesses relation "public.ct"  (seg0 slice1 172.18.0.3:7002 pid=50171)
```

Reply:
Do we need to? The issue is fairly straightforward and the same across all of them, I think there is no benefit for checking all of them separately

Reply:
I'm not sure, for my opinion tests should cover all threads in the code. But in current case this requires writing a lot of extra code. @whitehawk what do you think?

Reply:
I think it is better to add more test cases to avoid any possible breakages in future.

Reply:
Done